plugins/knife/knife-deployment-resize.rb

```diff
@@ -256,6 +260,13 @@ def run
 
         for s_node in s_nodes
           s_node.run_list << "role[shared_config]"
+
+          if config[:scscf_only]
+            s_node.set[:clearwater][:upstream_hostname] = "scscf.$sprout_hostname"
```

Comment:
It's a shared config setting, and we use the first Sprout to set up shared_config